### ISSUE TITLE
Error on export when writing into the db

### ISSUE THREAD

**Issue by schmunk42** [sequelpro/sequelpro#2152]
I had several write jobs on a MySQL database, which I also exported in parallel with SequelPRO.

This is my broken output snippet:

```
[...]
	(11201,147,14707120,NULL,3,'','','2014-06-06 00:00:00','2017-12-31 00:00:00',28.50,4.90,'',0.00,0,0,0.00,'L',1)11202,147,14707121,NULL,3,'','','2014-06-06 00:00:00','2017-12-31 00:00:00',28.50,4.90,'',0.00,0,0,0.00,'L',1),
	(;

/*!40000 ALTER TABLE `condition` ENABLE KEYS */;
UNLOCK TABLES;
[...]
```

MariaDB 10.1, running with Docker, SequelPro Client 1.0.2

**Comment by dmoagx** [comment]
Duplicate of #1726 
This is already fixed in 1.1rc1: http://www.sequelpro.com/blog/2015.06/sequel-pro-1-1-rc1/

**Comment by dmoagx** [comment]
OK, while I believe this might be the same issue please confirm to make sure.

**Comment by schmunk42** [comment]
Can't really reproduce it with 1.1-RC1, will reopen this issue if needed.
